tokens/package.json

```diff
@@ -24,7 +24,7 @@
 	"main": "dist/css/index.css",
 	"module": "style-dictionary.config.js",
 	"devDependencies": {
-		"@adobe/spectrum-tokens": "^13.10.1",
+		"@adobe/spectrum-tokens": "^13.11.0",
```

Comment:
There are some tokens from [13.12.0](https://github.com/adobe/spectrum-tokens/releases/tag/%40adobe%2Fspectrum-tokens%4013.12.0) that we could use to fix our accordion spacing, would it be possible to take in that version instead? I'm wondering if renovate didn't update it because of the manual changes on this PR?

Reply:
@rise-erpelding Yes, I'll make sure to bump that. The Token build is failing in after this v13.11.0 release because of the introduction of composite tokens.